Tools/autotest/arducopter.py

```diff
@@ -6606,9 +6606,9 @@ def fly_wind_baro_compensation(self):
             })
             self.reboot_sitl()
             self.set_parameters({
-                "EK3_DRAG_BCOEF_X": 361.000000,
-                "EK3_DRAG_BCOEF_Y": 361.000000,
-                "EK3_DRAG_MCOEF": 0.082000,
+                "EK3_DRAG_BCOEF_X": 91.2220,
```

Comment:
If this is in the parameter file, do we need it here?
Wouldn't it be more appropriate to be in the parameter file?

Reply:
I agree, seems odd having it hidden here.